### PR TITLE
`add-new-source.md` updates

### PR DESCRIPTION
**Pull request by joeflack4**
- Docs: Updated docs/developer/add-new-source.md: Created sub-sections, added some additional information, and updated step 1 info.

### CODE REVIEW

File: docs/developer/add-new-source.md [hunk 1]
```diff
@@ -1,6 +1,14 @@
 # How to add a new source to the Mondo Ingest
 
-1. Add a new import and component to [src/ontology/mondo-ingest-odk.yaml](https://github.com/monarch-initiative/mondo-ingest/blob/main/src/ontology/mondo-ingest-odk.yaml).
-2. Add a new metadata file to [src/ontology/metadata](https://github.com/monarch-initiative/mondo-ingest/blob/main/src/ontology/metadata). It is important we try and document as much about the source as we can.
-3. Update the Website Table of Contents [mkdocs.yaml](https://github.com/monarch-initiative/mondo-ingest/blob/main/mkdocs.yaml)
+## 1. `mondo-ingest-odk.yaml`
+### 1.1. Add source info
+Add a new filename and URL to the `products` > `components` section in [src/ontology/mondo-ingest-odk.yaml](https://github.com/monarch-initiative/mondo-ingest/blob/main/src/ontology/mondo-ingest-odk.yaml).
```
Comment (joeflack4) [inline docs/developer/add-new-source.md:5]: @matentzn Can you confirm that what I've written here looks like what we want? This previously said "add new import and component". The `imports` section of `mondo-ingest-odk.yaml` has RO and OMO in it, whereas the `products.components` section has what I'm more familiar with.

File: docs/developer/add-new-source.md [hunk 2]
```diff
@@ -1,6 +1,14 @@
...
 
+### 1.2. `update_repo`
+From `src/ontology`, run `sh run.sh make update_repo`.
```
Comment (joeflack4) [inline docs/developer/add-new-source.md:8]: @matentzn Added this next step about `update_repo`

File: docs/developer/add-new-source.md [hunk 3]
```diff
@@ -1,6 +1,14 @@
...
+
+## 2. `metadata/*.yaml`
+Add a new metadata file to [src/ontology/metadata](https://github.com/monarch-initiative/mondo-ingest/blob/main/src/ontology/metadata). It is important we try and document as much about the source as we can.
+
+## 3. `mkdocs.yaml`
```
Comment (joeflack4) [inline docs/developer/add-new-source.md:13]: @matentzn Do you think something is missing here? I noticed that it looks like I manually needed to add links to `gard.md` in several places. It doesn't look like `mkdocs build` is what I want to run. Perhaps we need to write something here about manually making some other additions? Such as:

1. `sources/ONTOLOGY.md`
2. Adding ONTOLOGY to `sources.md`
3. What about `metrics/ONTOLOGY.md` as well as adding a link to that file in `mkdocs.yaml` as well?

**Review by joeflack4 — COMMENTED** [review]
@hrsdhgd @souzadevinicius FYI / if you have time to review

File: docs/developer/add-new-source.md (hunk 2 above)
Comment (matentzn) [reply docs/developer/add-new-source.md:8]: Yes, good!

File: docs/developer/add-new-source.md (hunk 1 above)
Comment (matentzn) [reply docs/developer/add-new-source.md:5]: component is correct, so what you have now is right

File: docs/developer/add-new-source.md (hunk 3 above)
Comment (matentzn) [reply docs/developer/add-new-source.md:15]: Unfortunately ues, mkdocs.yaml is entirely automatically managed. So all of these need to be added.

**Review by matentzn — APPROVED** [review]

**Comment by souzadevinicius** [comment]
> @hrsdhgd @souzadevinicius FYI / if you have time to review

LGTM

File: docs/developer/add-new-source.md (hunk 2 above)
Comment (joeflack4) [reply docs/developer/add-new-source.md:8]: I added a note about it needing to run twice as well. However, as I mentioned in [this comment](https://github.com/monarch-initiative/mondo-ingest/issues/308#issuecomment-1550456717), running it a second time did not seem to do anything for me.

File: docs/developer/add-new-source.md (hunk 3 above)
Comment (joeflack4) [reply docs/developer/add-new-source.md]: Np. I added instructions for adding these. I see now that there are some goals that help with these:
`../../docs/sources/%.md`
`../../docs/metrics/%.md`

**Pull request merged**